### ISSUE TITLE
add_velocity levels feature

### ISSUE THREAD

**Issue by StevePny** [giswqs/leafmap#234]
### Environment Information

-   leafmap version: master branch, post 0.9.1 
-   Python version: 3.9
-   Operating System: mac osx

### Description

The functionality I needed was aded recently (3 days ago, after v0.9.1). At the moment I've just copied the two modified files to replace my local copies. There's a piece of code in leafmap/leafmap.py in the add_velocity method that has a bug:

Currently it has:

```
if level_dimension in coords:
      ds = ds.isel(lev=level_index, drop=True)
```

The name of my vertical coordinate in the netcdf file is "plev" and not "lev". The argument sent in is level_dimension='plev'. 

But to make it general it should be something more like this (though this doesn't actually work):
```
if level_dimension in coords:
      ds = ds.isel(eval(level_dimension)=level_index, drop=True)
```

I'm not sure of the best way to do this, but at the moment the intended functionality doesn't work.

### What I Did

To make this work for my application at the moment I have just done a hardcoded change to the source code:
```
if level_dimension in coords:
      ds = ds.isel(plev=level_index, drop=True)
```

**Comment by giswqs** [comment]
You are welcome to submit a pull request

**Comment by giswqs** [comment]
I am not quite sure how to resolve this issue. The `lev` cannot be hardcoded. Since the `add_velocity()` function accepts xarray Dataset, you can preprocess the data if needed and then pass it to the `add_velocity()` function. If you have a better solution, please feel free to submit a pull request. 

```python
ds = xr.open_dataset(data)
ds = ds.isel(plev=level_index, drop=True)
m.add_velocity(data=ds, zonal_speed, meridional_speed)
```

**Comment by StevePny** [comment]
I do still think this is a bug. It is already hardcoded as "lev", even though the name "level_dimension" is supposed to be provided as an input. The functionality is not working as intended.

**Comment by giswqs** [comment]
@StevePny I just committed code to fix this. Please update the package using `leafmap.update_package()` and try again. 

```python
params = {level_dimension: level_index}
if level_dimension in coords:
    ds = ds.isel(drop=True, **params)
````

**Comment by mathomp4** [comment]
Yeah...I'm not the best at Python. I was afraid I'd coded it in a "works for me" way. Thanks @giswqs for making it better Python! 😄

**Comment by giswqs** [comment]
@StevePny @mathomp4 Thank you both for your contribution that makes leafmap better!
